src/map/mail.cpp

```diff
@@ -418,3 +419,39 @@ void mail_refresh_remaining_amount( struct map_session_data* sd ){
 		sc_start2( &sd->bl, &sd->bl, SC_DAILYSENDMAILCNT, 100, doy, 0, -1 );
 	}
 }
+
+void npc_mail_send(struct npc_data *nd, const char *dest_name, const char *title, const char *body_msg, int body_len, int zeny) {
```

Comment:
1. Why VOID? Why not to try to use ENUMs with named constants which will signal that something wrong?
2. suggestion: const char body_msg to just msg, and body_len to msg_len